buildspec.test.yml

```diff
@@ -25,6 +25,13 @@ phases:
 
   post_build:
     commands:
+      - |
+        echo -e "\n\nSend alert if the tests have failed\n"
+        echo "Debug output:"
+        echo "CODEBUILD_BUILD_SUCCEEDING: ${CODEBUILD_BUILD_SUCCEEDING}"
+        echo "git branch: $(git branch)"
+        echo "git branch --contains \$CODEBUILD_RESOLVED_SOURCE_VERSION: $(git branch --contains $CODEBUILD_RESOLVED_SOURCE_VERSION)"
+        echo "git branch --contains \$CODEBUILD_RESOLVED_SOURCE_VERSION | grep -q "main": $(git branch --contains $CODEBUILD_RESOLVED_SOURCE_VERSION | grep -q "main" || true)"
```

Comment:
To help us debug why alerts are not always sent for failures on the main branch.